Calculation.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```

Comment:
Always remove unused 'usings'. Only leave code that's actually being used. You want to be pedantic about readability.